ui/StatusQ/src/StatusQ/Components/StatusDraggableListItem.qml

```diff
@@ -180,18 +190,22 @@ ItemDelegate {
 
     background: Rectangle {
         color: "transparent"
-        border.width: 1
+        border.width: customizable ? 0 : 1
         border.color: Theme.palette.baseColor2
-        radius: 8
+        radius: customizable ? 0 : 8
```

Comment:
This can go too, if it's invisible, the radius doesn't matter

Reply:
true but we don't have to paint rounded corners either, wdyth